Packs/CommonScripts/Scripts/FileCreateAndUploadV2/FileCreateAndUploadV2.py

```diff
@@ -1,27 +1,44 @@
 import demistomock as demisto  # noqa: F401
 from CommonServerPython import *  # noqa: F401
 import base64
+from pathlib import Path
+from typing import Any
+
+
+def get_data_from_file(entry_id: str):
+    """
+    This function reads the file associated with the entry_id and returns its data as bytes.
+    """
+    try:
+        data = Path(demisto.getFilePath(entry_id)['path']).read_bytes()
+    except Exception as e:
+        raise DemistoException(f'There was a problem opening or reading the file.\nError is: {e}')
+    return data
+
+
+def decode_data(data: Any, data_encoding: str):
+    """
+    Given data and its encoding, this function decodes the data according to the provided encoding and returns it.
+    """
+    if data_encoding == 'base64':
+        data = base64.b64decode(data)
+    elif data_encoding != 'raw':
+        raise ValueError(f'Invalid data encoding name: {data_encoding}')
+    return data
```

Comment:
to make it easier to read, let's do `if b64: return ... elif raw: return ... <end of if> raise`